### PR TITLE
Red 3.4.3 - Changelog

### PR DESCRIPTION
**Pull request by Jackenmen**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [ ] New feature
- [x] Docs

### Description of the changes
Changelog for Red version 3.4.3.

Rendered documentation can be seen at https://red-discordbot--4613.org.readthedocs.build/en/4613/changelog_3_4_0.html#redbot-3-4-3-2020-11-16

### CODE REVIEW

File: docs/changelog_3_4_0.rst [hunk 1]
```diff
@@ -1,5 +1,75 @@
...
+
+    Red 3.4.3 uses a new Lavalink jar that you will need to manually update from `our GitHub <https://github.com/Cog-Creators/Lavalink-Jars/releases/tag/3.3.1.4_1132>`__.
+
+End-user changelog
+------------------
+
+Core Bot
+********
+
+- Added ``[p]set competing`` command that allows users to set bot's competing status (:issue:`4607`, :issue:`4609`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Added ``[p]set competing`` command that allows users to set the bot's competing status (:issue:`4607`, :issue:`4609`)
```

File: docs/changelog_3_4_0.rst [hunk 2]
```diff
@@ -1,5 +1,75 @@
...
+
+Audio
+*****
+
+- Volume changes on ARM systems running a 64 bit OS will now work again (:issue:`4608`)
+- Fixed only 100 results being returned on a Youtube playlist (:issue:`4608`)
+- Fixed YouTube VOD duration set to unknown (:issue:`3885`, :issue:`4608`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Fixed YouTube VOD duration being set to unknown (:issue:`3885`, :issue:`4608`)
```

File: docs/changelog_3_4_0.rst [hunk 3]
```diff
@@ -1,5 +1,75 @@
...
+- Fixed some YouTube livestreams getting stuck (:issue:`4608`)
+- Fixed internal Lavalink manager failing for Java with untypical version formats (:issue:`4608`)
+- Improved AAC audio handling (:issue:`4608`)
+- Added support for SoundCloud HLS streams (:issue:`4608`)
+
+Economy
+*******
+
+- ``[p]leaderboard`` command no longer fails in DMs when global bank is used (:issue:`4569`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The ``[p]leaderboard`` command no longer fails in DMs when a global bank is used (:issue:`4569`)
```

File: docs/changelog_3_4_0.rst [hunk 4]
```diff
@@ -1,5 +1,75 @@
...
+
+Mod
+***
+
+- Ban reason is now properly set in audit log and modlog when using ``[p]massban`` command (:issue:`4575`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The ban reason is now properly set in the audit log and modlog when using the ``[p]massban`` command (:issue:`4575`)
```

File: docs/changelog_3_4_0.rst [hunk 5]
```diff
@@ -1,5 +1,75 @@
...
+- ``[p]userinfo`` command now shows the new Competing activity (:issue:`4610`, :issue:`4611`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The ``[p]userinfo`` command now shows the new Competing activity (:issue:`4610`, :issue:`4611`)
```

File: docs/changelog_3_4_0.rst [hunk 6]
```diff
@@ -1,5 +1,75 @@
...
+
+Modlog
+******
+
+- ``[p]case`` and ``[p]casesfor`` commands no longer fail when the bot doesn't have Read Message History permission in the modlog channel (:issue:`4587`, :issue:`4588`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The ``[p]case`` and ``[p]casesfor`` commands no longer fail when the bot doesn't have Read Message History permission in the modlog channel (:issue:`4587`, :issue:`4588`)
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
😃

**Review by Flame442 — APPROVED** [review]
🎉

**Review by bobloy — APPROVED** [review]

**Review by Vexed01 — APPROVED** [review]
I think I smell another landslide of approvals coming 😏

**Review by NeuroAssassin — APPROVED** [review]

**Review by phenom4n4n — APPROVED** [review]

**Comment by Jackenmen** [comment]
Not this time...

**Pull request merged**